src/main/java/hw_17/Test4.java

```diff
@@ -0,0 +1,29 @@
+package hw_17;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.junit.Assert;
+
+import java.time.Duration;
+
+public class Test4 {
+    public static void main(String[] args) {
+        WebDriver driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://makeup.com.ua/ua/");
+
+        WebDriverWait wait = new WebDriverWait(driver, Duration.ofSeconds(15));
+
+        WebElement hairButton = wait.until(ExpectedConditions.elementToBeClickable(By.cssSelector("a.menu-list__link[href='/ua/categorys/20272/']\n")));
```

Comment:
1) why do you use "\n" in the end of selector?
2) Are you sure you need a waiter for this element?

Reply:
done